.github/workflows/pull-request.yaml

```diff
@@ -93,7 +93,7 @@ jobs:
       - name: Run cargo-tarpaulin
         uses: actions-rs/tarpaulin@v0.1
         with:
-          args: "-- --test-threads 1"
+          args: "--workspace --exclude prql-python -- --test-threads 1"
```

Comment:
Very low priority but if we can add a TODO to put this in a config file, that'd be great